.github/file-filters.yml

```diff
@@ -0,0 +1,6 @@
+backend:
```

Comment:
Q: Should we skip an entire job by creating a proxy one instead like one dorny's examples
https://github.com/dorny/paths-filter#examples
Or it is better we do step by step checking?

Reply:
I think the step method looks optimal , without another CI runner,I think it better like this